src/core_plugins/kibana/public/management/index.js

```diff
@@ -48,6 +48,10 @@ uiModules
       if ($scope.section) {
         $scope.section.items.forEach(item => {
           item.active = `#${$location.path()}`.indexOf(item.url) > -1;
+
+          if ($scope.sectionName === 'kibana' && item.id === 'indices' && item.active) {
+            $scope.omitPages = ['indices'];
+          }
```

Comment:
This is better than hardcoding in the template, but it still doesn't feel like the right place (the hardcoding has moved from the template to the controller, but it is still part of the generic `kbnManagementApp` directive).

The `kbnManagementApp` directive is meant to be reusable and is indeed widely used across several places in the codebase. For example, https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/public/management/sections/indices/_field_editor.html#L1 or https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/public/management/sections/settings/index.html#L1 or https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/public/management/sections/indices/_edit.html#L1. It could even be used by 3rd party plugins to Kibana that wish to register additional sections under Management. So it doesn't feel right to me that logic about a omitting pages for a specific section should be baked in the definition of this generic directive itself. I would like to move this responsibility closer to where those specific sections use the `kbnManagementApp` directive.